modules/services/kanshi.nix

```diff
@@ -0,0 +1,201 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.kanshi;
+
+  outputModule = types.submodule {
+    options = {
+
+      criteria = mkOption {
+        type = types.str;
+        description = ''
+          <literallayout>
```

Comment:
Don't use `literallayout`, you don't know there this text will be displayed so forcing a specific layout of plain paragraphs will almost certainly look bad.